keel-sql/keel-sql.gradle.kts

```diff
@@ -1,16 +1,22 @@
 import ch.ayedo.jooqmodelator.gradle.JooqModelatorTask
 import com.diffplug.gradle.spotless.SpotlessExtension
 
+val buildingInDocker = project.properties["buildingInDocker"]?.toString().let { it == "true" }
+
 plugins {
   `java-library`
   id("kotlin-spring")
   id("ch.ayedo.jooqmodelator") version "3.9.0"
+  id("nu.studer.jooq") version "5.0.1"
   id("org.liquibase.gradle") version "2.0.4"
 }
 
 afterEvaluate {
-  tasks.getByName("compileKotlin") {
-    dependsOn("generateJooqMetamodel")
+  // When not running in Docker, we can use jooqModelator (which itself uses Docker) to generate the jOOQ meta-model
+  if (!buildingInDocker) {
+    tasks.getByName("compileKotlin") {
+      dependsOn("generateJooqMetamodel")
+    }
```

Comment:
Why don't we just switch over completely?

Reply:
We can... I was just not looking to spend too much more time on this at this point as jooqModelator works fine for our use. We'd need to add another task to spin up the docker container, etc.

Reply:
Let me give it a try -- if I can use the vanilla mysql container, then it should be relatively straightforward.